modules/cpmso-feat-replace.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control_plane_machine_management/cpmso-using.adoc
+
+:_content-type: PROCEDURE
+[id="cpmso-feat-replace_{context}"]
+= Replacing a control plane machine
```

Comment:
The concept of getting a control plane machine replaced is also true for the `RollingUpdate` strategy.

If you do the following steps with `RollingUpdate` the Operator still creates the required replacement machine.

So we could add something along the lines that independenty from the update strategy the user can follow these steps to get a machine replaced.